website/content/docs/k8s/installation/install.mdx

```diff
@@ -203,7 +107,40 @@ NAME: consul
 ...
 ```
 
-#### Enable Consul Service Mesh on select namespaces
+### Enable the Consul CNI plugin
+
+By default, Consul generates a `connect-inject init` container as part of the Kubernetes pod startup process when Consul is in [transparent proxy mode](/docs/connect/transparent-proxy). The container configures traffic redirection in the service mesh through the sidecar proxy. To configure redirection, the container requires elevated CAP_NET_ADMIN privileges, which may not be compatible with security policies in your organization. 
+
+Instead, you can enable the Consul container network interface (CNI) plugin to perform traffic redirection. Because the plugin is executed by the Kubernetes kubelet, the plugin already has the elevated privileges necessary to configure the network. 
+
+Add the following configuration to your `config.yaml` file to enable the Consul CNI plugin:
+
+<CodeBlockConfig filename="config.yaml">
+
+```yaml
+global:
+  name: consul
+connectInject:
+  enabled: true
+  cni:
+    enabled: true
+    logLevel: info
+    cniBinDir: "/opt/cni/bin"
+    cniNetDir: "/etc/cni/net.d"
+```
```

Comment:
Is it possible to include a code tab here, so we can display GKE here as another config? Perhaps name the first Reference config and the second GKE? The GKE config is below:

```yaml
 global:
   name: consul
 connectInject:
   enabled: true
   cni:
     enabled: true
     logLevel: info
      cniBinDir: "/home/kubernetes/bin"
     cniNetDir: "/etc/cni/net.d"
 ```